### DISCUSSION TITLE
Welcome to Wire Elements Modal Discussions!

### DISCUSSION THREAD

**Discussion by PhiloNL** [wire-elements/modal#87] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by rik43** [comment]
hello, are there any plans to make an offcanvas?
https://getbootstrap.com/docs/5.1/components/offcanvas/

**Reply by PhiloNL** [reply]
Maybe, but it will probably be with TailwindCSS instead of Bootstrap.
